ledger/participant-state/kvutils/src/main/protobuf/com/daml/ledger/participant/state/kvutils/store/state.proto

```diff
@@ -54,12 +54,13 @@ message DamlCommandDedupKey {
 
 message DamlCommandDedupValue {
   reserved 1; // was record_time
+  reserved 3; // was submission_time
   oneof time {
     // The time until when future commands with the same
     // deduplication key will be rejected due to a duplicate submission.
     // Read-only for backwards compatibility
     google.protobuf.Timestamp deduplicated_until = 2 [deprecated = true];
-    google.protobuf.Timestamp record_time = 3;
+    google.protobuf.Timestamp record_time = 5;
```

Comment:
1 used to be `record_time`. Can we reuse it or is that a terrible idea?

Reply:
I didn't really check what type we had. But I would prefer to continue clean and use a new index just to be on the safe side. What's deleted remains deleted.

Reply:
It's a terrible idea and reusing it merely gets rid of 1 line of unused protobuf, but brings the potential to cause issues. The clean way out is to never reuse a previously used field.

Reply:
I thought you might say that.

Reply:
Also, see my https://github.com/digital-asset/daml/pull/11780#discussion_r752431994 in the other PR. We probably don't even need this change.

Reply:
Alright, closing this one and purging the field in the other PR.